e2e/test/prerequisites/E2ETestsSetup/e2eTestsArmTemplate.json

```diff
@@ -36,45 +89,54 @@
         },
         "HubName": {
             "type": "string",
+            "defaultValue": "[resourceGroup().name]",
             "metadata": {
                 "description": "The name of the main IoT hub used by tests."
             }
         },
         "FarHubName": {
             "type": "string",
+            "defaultValue": "[concat(resourceGroup().name, 'Far')]",
             "metadata": {
                 "description": "The name of the far away IoT hub used by tests."
             }
         },
         "DeviceProvisioningServiceName": {
             "type": "string",
+            "defaultValue": "[resourceGroup().name]",
             "metadata": {
                 "description": "The name of the DPS used by tests."
             }
-        },        
+        },
         "DpsCustomAllocatorFunctionName": {
             "type": "string",
-            "defaultValue": "DpsCustomAllocator"
+            "defaultValue": "DpsCustomAllocator",
```

Comment:
I think we should actually call this one `DpsCustomAllocatorByHubLength`, because that is the logic it uses. If we ever chose to add more custom allocators, we'd want to distinguish them.